lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -1333,9 +1333,9 @@ def to_id(name):  # pylint: disable=missing-docstring
             discussion_topics=["General", "Feedback"],
             cohorted_discussions=["Feedback", "random_inline"]
         )
-        self.assertTrue(
+        self.assertFalse(
             utils.is_commentable_cohorted(course.id, to_id("random")),
-            "By default, Non-top-level discussion is always cohorted in cohorted courses."
+            "By default, Non-top-level discussion are not cohorted in a cohorted courses."
```

Comment:
tiny nit: "in a cohorted courses" could be "in cohorted courses" or "in a cohorted course"